packages/jest-config/src/index.js

```diff
@@ -24,6 +24,7 @@ import readConfigFileAndSetRootDir from './read_config_file_and_set_root_dir';
 export {getTestEnvironment, isJSONString} from './utils';
 export {default as normalize} from './normalize';
 export {default as deprecationEntries} from './deprecated';
+export {_replaceRootDirInPath} from './utils';
```

Comment:
This should be renamed to `replaceRootDirInPath`

Reply:
You're right 😄